### ISSUE TITLE
Rename the rename option

### ISSUE THREAD

**Issue by haltlos** [cryptomator/ios#254]
### Please agree to the following

- [X] I have searched [existing issues](https://github.com/cryptomator/cryptomator/issues?q=) for duplicates
- [X] I agree to follow this project's [Code of Conduct](https://github.com/cryptomator/cryptomator/blob/develop/.github/CODE_OF_CONDUCT.md)

### Summary

The option to rename a vault needs to be rephrased

### Motivation

Currently, on iOS and iPadOS, a Dropbox vault can be renamed. The option to rename a vault is already familiar from the macOS version of Cryptomator. Unfortunately, the developers completely overlook that renaming a folder and renaming a vault (mount) are two different things. On macOS, the option to name a vault does not change the folder name itself in in the cloud. However, on iOS the option to rename DOES rename the folder!

You should clarify WHAT is being renamed. Just using the word "Rename" is not clear. Use "Rename folder" to clarify that the folder itself gets renamed.

**Comment by tobihagemann** [comment]
Duplicate of #177

**Comment by haltlos** [comment]
This is not a duplicate of #177.

**Comment by tobihagemann** [comment]
Then I misunderstood your feature request. Didn't you suggest a rename function in the iOS app that lets you rename the vault without renaming the folder?

**Comment by haltlos** [comment]
Yes, I was merely suggesting to change the word "Rename to "Rename folder" to clarify what gets renamed. :)
